UI-3053: Added Smart PBX SIP Device feature key drag and drop reordering

AnthonyPerish opened the pull request:
UI-3053 - This addition allows for reordering of feature keys while editing a device from Smart PBX.  This utilizes JQuery sortable and resaves the data in the specified order when saving.

azefiel commented on submodules/devices/devices.js:
```diff
@@ -337,6 +337,20 @@ define(function(require) {
 					});
 				});
 
+				$(function() {
```
The DOM is ready to be manipulated at this point, there is no need to wrap your event handler in such a function.

AnthonyPerish replied:
Thank you, you are correct.  I'll go ahead and fix this.

azefiel commented on submodules/devices/devices.js:
```diff
@@ -337,6 +337,20 @@ define(function(require) {
+									});
+						}
+					});
+					templateDevice.find('.keys').disableSelection();
```
I don't see the point of using this method, could you explain why this is needed?

AnthonyPerish commented:
Thank you for the comments, I have gone ahead and removed the extra function around this code and removed the disableSelection code.  You were correct that it was not necessary in this instance. 

Please review and let me know if you have any other questions or comments.
